lib/src/main/java/com/auth0/android/lock/events/DatabaseSignUpEvent.java

```diff
@@ -35,33 +35,18 @@
 
 import java.util.Map;
 
-public class DatabaseSignUpEvent {
+public class DatabaseSignUpEvent extends DatabaseEvent {
```

Comment:
What we get from extending this object? (besides the properties email, username and password)